systemtests/network_test.go

```diff
@@ -279,11 +296,21 @@ func (s *systemtestSuite) testNetworkAddDeleteTenant(c *C, encap string) {
 			tenantNames[tenantName] = []string{}
 
 			for networkNum := 0; networkNum < numContainer/len(s.nodes); networkNum++ {
+				var v6subnet, v6gateway string
+				if networkNum%2 == 0 {
+					v6subnet = ""
+					v6gateway = ""
+				} else {
+					v6subnet = fmt.Sprintf("1001:%d:%d::/120", tenantNum, networkNum)
+					v6gateway = fmt.Sprintf("1001:%d:%d::254", tenantNum, networkNum)
+				}
```

Comment:
It'll be good to add similar IPv6 subnets into some of the tests in trigger_test.go as well. They mainly test netplugin/netmaster restarts, disconnects etc. This will excersize IPv6 restartability and route distribution failure scenarios. This will test the new IPv6 related code in ofnet